### PR TITLE
ALL : Maintenance release

### LINKED ISSUE #6129: Add: Contacto Bolivia
### Channel Name

Contacto Bolivia

### Country

Bolivia

### Language

Spanish

### Website

https://www.contactobolivia.com.bo/

### Stream URL

N/A

### Notes

Stream tokenized

### LINKED ISSUE #6794: Add: California Music Channel (CMC-TV)
### Channel Name

California Music Channel (CMC-TV)

### Channel ID

CaliforniaMusicChannel.us

### Website

https://cmc-tv.com/

### Stream URL

https://cmc-ono.amagi.tv/hls/amagi_hls_data_cmcAAAAAA-cmc-ono/CDN/playlist.m3u8?device[did]=%7BPSID%7D&device[dnt]=%7BDID%7D&us_privacy=%7BUS_PRIVACY%7D&coppa=%7BCOPPA%7D

### Notes

I think, above stream URL has to be used in place of https://github.com/iptv-org/iptv/blob/66892286541ac904d0993e52cbf750ab1de024d9/streams/us.m3u#L327

1. As per streamtest.in, it doesn't seem to be Geo-blocked
    1. https://streamtest.in/runtest/FpTxIEJAb5ejHCS0oWHxPg Singapore
    2. https://streamtest.in/runtest/q7BmLbAbII0JFQgKijTQzQ USA
3. It is having `360, 504, 720, 1080` resolutions

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #6847: Add: TCM Latino | (México) Or (Argentina)
### Channel Name

TCM

### Channel ID

TCMArgentina.us

### Website

photocall.tv 

### Stream URL

https://cdn.prepro.cvattv.com.ar/live/vod/TCM/SA_Live_WebVTT_FTA/TCM.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #6970: Add: Bang Bang
### Channel Name

Bang Bang

### Channel ID

_No response_

### Website

https://www.digitalb.al/

### Stream URL

http://199.127.60.117:8081/live/bangbang/playlist.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #7000: Add: MNB Sport
### Channel Name

MNB Sport

### Channel ID

MNBSport.mn

### Website

https://www.mnb.mn/live/sport

### Stream URL

https://live.mnb.mn/live/mnb_sport.stream/playlist.m3u8

### Notes

Not 24/7

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #7001: Add: MNB Family
### Channel Name

MNB Гэр бүл

### Channel ID

_No response_

### Website

https://www.mnb.mn/live/family

### Stream URL

https://live.mnb.mn/live/mnb_family.stream/playlist.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #7002: Add: TV5 Mongolia
### Channel Name

TV5

### Channel ID

TV5.mn

### Website

http://tv5.mn/index.php/live

### Stream URL

http://d1qwew1jz60jqg.cloudfront.net/MYASIATV/TV5/playlist.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #7063: Add: Once Niñas Y Niños 24/7 | (Mexico)
### Channel Name

Once Niñas Y Niños [XEIPN-TDT 11.2]

### Channel ID

_No response_

### Website

_No response_

### Stream URL

http://jjooliveclarovideo.akamaized.net/Content/hls/Live/Channel(tveducationch01)/Stream(05)/index.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #7077: Add: Balapan
### Channel Name

Balapan

### Channel ID

Balapan.kz

### Website

https://balapan.tv/live

### Stream URL

https://balapantv-stream.qazcdn.com/balapantv/balapantv/playlist.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #7078: Add: Qazaqstan (International Feed)
### Channel Name

Qazaqstan

### Channel ID

Qazaqstan.kz

### Website

https://qazaqstan.tv/live

### Stream URL

https://qazaqstantv-stream.qazcdn.com/international/international/playlist.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #7079: Add: Abai TV
### Channel Name

Abai TV

### Channel ID

AbaiTV.kz

### Website

https://abaitv.kz/kz/live

### Stream URL

https://abaitv-stream.qazcdn.com/abaitv/abaitv/playlist.m3u8

### Notes

Kazakhstan

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #7119: Add:BBC WORLD NEWS
### Channel Name

BBC WORLD NEWS 

### Channel ID

_No response_

### Website

_No response_

### Stream URL

[_No response_
](http://103.199.161.254/Content/bbcworld/Live/Channel(BBCworld)/index.m3u8
)
### Notes

I have BBC WORLD NEWS (LOW QUALITY) but it is low quality. Looking for high quality. 

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #7163: Add: [Tring] 3Plus
### Channel Name

3Plus

### Channel ID

_No response_

### Website

_No response_

### Stream URL

http://199.127.60.117:8081/live/tring3plus/playlist.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #7234: Add: Viasat History
### Channel Name

Viasat History

### Channel ID

_No response_

### Website

_No response_

### Stream URL

https://sc.id-tv.kz/ViasatHistory_34_35.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #7235: Add: Canal Once (Mexico) [XEIPN]
### Channel Name

Canal Once [XEIPN-TDT 11.1]

### Channel ID

_No response_

### Website

_No response_

### Stream URL

https://d24sa4vr9gvjv.cloudfront.net/index.m3u8

### Notes

This Is Connected To Issue #7063 

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #7257: Add: WBRZ+ Weather Channel | (US)
### Channel Name

WBRZ Weather Channel 

### Channel ID

_No response_

### Website

_No response_

### Stream URL

https://d2a6g6zgxepbwi.cloudfront.net/wbrz-edge/weather.stream_360p/playlist.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #7294: Add:  Noovo (French Canadian TV channel)
### Channel Name

Noovo

### Channel ID

Noovo.ca

### Website

https://www.noovo.ca/

### Stream URL

https://pe-fa-lp03a.9c9media.com/live/NOOVO/p/hls/00000201/716cf4c845225692/manifest.m3u8

### Notes

Play fine in Tivimate + PerfectPlayer (both Audio **AND** video)

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #7297: Add: YouTV
### Channel Name

YouTV

### Channel ID

_No response_

### Website

https://tv.sikkylab.me

### Stream URL

https://sikkylab.me/tv/stream.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### PR DESCRIPTION
**Pull request by BellezaEmporium**
Gets all valid stream proposals up in our playlists.

BO
Close #6129 

MX
Close #7235 
Close #7063

NO
Kazakhstan server + added more channels than asked.

Close #7234

CA
Close #7294

US
Close #7297 
Close #7257 
Close #6847 
Close #6794 

UK
Close #7119

KZ
Close #7079
Close #7078 
Close #7077

AL
Close #7163 
Close #6970

MN
Close #7002
Close #7001 
Close #7000

### CODE REVIEW

**Review by its0x08 — DISMISSED** [review]

**Comment by LaneSh4d0w** [comment]
let's push this for now, second batch coming afterwards.

File: streams/mn.m3u [hunk 1]
```diff
@@ -1,5 +1,11 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="MongolynMedee.mn" status="online",Монголын Мэдээ (576p)
-http://103.14.38.107:1935/live/mn2.stream/chunklist.m3u8
+http://103.14.38.107:1935/live/mn2.stream/playlist.m3u8
 #EXTINF:-1 tvg-id="MUONT1.mn" status="online",МҮОНТ-1 (576p)
-http://103.14.38.107:1935/live/mnb.stream/chunklist.m3u8
+http://103.14.38.107:1935/live/mnb.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="TV5.mn" status="online",TV5
+http://d1qwew1jz60jqg.cloudfront.net/MYASIATV/TV5/playlist.m3u8
```
Comment (freearhey) [inline streams/mn.m3u:7]: image

File: streams/pr.m3u [hunk 2]
```diff
@@ -15,3 +15,5 @@ https://59514edd5dd8e.streamlock.net/unoradio/unoradio1/playlist.m3u8
 https://video1.getstreamhosting.com:1936/8212/8212/playlist.m3u8
 #EXTINF:-1 tvg-id="" status="error",Telemundo WKAQ Noticias Puerto Rico (1080p) [Not 24/7]
 https://wkaqlive-lh.akamaihd.net/i/PR_STREAM1@311877/master.m3u8
+#EXTINF:-1 tvg-id="" status="",Wipr
+https://streamwipr.pr/hls/wipr/playlist.m3u8
```
Comment (freearhey) [inline streams/pr.m3u:19]: image

File: streams/us.m3u [hunk 3]
```diff
@@ -2045,3 +2043,15 @@ https://www.livedoomovie.com/02_AXNHD_720p/chunklist.m3u8
...
+#EXTINF:-1 tvg-id="" status="",WBRZ Weather Channel Baton Rouge LA
+https://d2a6g6zgxepbwi.cloudfront.net/wbrz-edge/weather.stream_360p/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="",PHX TV (Phoenix TV)
+https://granicusliveus8-a.akamaihd.net/phoenix/G1498_002/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="",TCM Latinoamérica
+https://cdn.prepro.cvattv.com.ar/live/vod/TCM/SA_Live_WebVTT_FTA/TCM.m3u8
+#EXTINF:-1 tvg-id="" status="",CMC (California Music Channel)
+https://cmc-ono.amagi.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="",CBS KIRO-TV 7 Seattle WA
+https://amg00327-coxmediagroup-kiro7seattle-ono-4z7u9.amagi.tv/playlist.m3u8
```
Comment (freearhey) [inline streams/us.m3u:2057]: image

**Review by freearhey — CHANGES_REQUESTED** [review]

File: streams/us.m3u (hunk 3 above)
Comment (LaneSh4d0w) [reply streams/us.m3u:2057]: geolock

File: streams/pr.m3u (hunk 2 above)
Comment (LaneSh4d0w) [reply streams/pr.m3u:19]: it seems they've changed their address. going to address it.

File: streams/mn.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply streams/mn.m3u:7]: works for me via browser extension
snapshot

File: streams/pr.m3u (hunk 2 above)
Comment (LaneSh4d0w) [reply streams/pr.m3u:19]: https://streannliveprtv.cachefly.net/prtv1/prtv1/playlist.m3u8

**Review by freearhey — APPROVED** [review]

**Review by what-a — APPROVED** [review]

**Review by Central302 — APPROVED** [review]

**Pull request merged**